client_plugin/drivers/vfile/vfile_driver.go

```diff
@@ -568,6 +674,25 @@ func (d *VolumeDriver) MountVolume(name string, fstype string, id string, isRead
 		return "", errors.New(msg)
 	}
 
+	// mount is successful, update the clientList
```

Comment:
Inside mountVFileVolume, there is already code to call GetSwarmInfo to get the IP address. Can we move the following part of the new code to the end of mountVFileVolume function and merge with existing code there? So we can avoid calling GetSwarmInfo two times.